### PR TITLE
Update libp2p to v0.0.2

### PR DESCRIPTION
**Pull request by Harrm**
* I've followed [this guide](https://hunter.readthedocs.io/en/latest/creating-new/update.html)
  step by step carefully. **Yes**

---

---

### CODE REVIEW

File: cmake/projects/libp2p/hunter.cmake [hunk 1]
```diff
@@ -32,6 +32,17 @@ hunter_add_version(
...
 
+hunter_add_version(
+    PACKAGE_NAME
+    libp2p
+    VERSION
+    0.0.2
+    URL 
+    "https://github.com/soramitsu/cpp-libp2p/archive/v0.0.2.tar.gz"
+    SHA1
+    6501ce6cae5309a5597891b44d75de3ebabaf99e
```
Comment (igor-egorov) [inline cmake/projects/libp2p/hunter.cmake:43]: 197bf8a8ed89b2cbdc5ccbdc8ec01be23ee5d2f7

**Review by igor-egorov — CHANGES_REQUESTED** [review]

File: cmake/projects/libp2p/hunter.cmake (hunk 1 above)
Comment (igor-egorov) [reply cmake/projects/libp2p/hunter.cmake:43]: https://emn178.github.io/online-tools/sha1_checksum.html

Comment (xDimon) [reply cmake/projects/libp2p/hunter.cmake:43]:
```shell
Mi[15:03]~> wget -q https://github.com/soramitsu/cpp-libp2p/archive/v0.0.2.tar.gz
Mi[15:03]~> sha1sum v0.0.2.tar.gz 
197bf8a8ed89b2cbdc5ccbdc8ec01be23ee5d2f7  v0.0.2.tar.gz
```

Comment (xDimon) [inline cmake/projects/libp2p/hunter.cmake:43]:
```suggestion
    197bf8a8ed89b2cbdc5ccbdc8ec01be23ee5d2f7
```

**Review by xDimon — CHANGES_REQUESTED** [review]

**Review by xDimon — APPROVED** [review]

**Review by igor-egorov — APPROVED** [review]

**Pull request merged**